backward_paper/runner.py

```diff
@@ -68,46 +68,53 @@ def __init__(self):
         self.sandbox = True
         self.fig_name = None
         self.return_to_Q0 = False
+        self.ekos = []
+        self.intermediate_Q = None
 
-    def run_me(self, theory, ocard, _pdf):
-
-        # Go back to the inital scale?
+    def run(self, theory_updates, ocard_updates, pdfs, use_replicas=False):
         if self.return_to_Q0:
```

Comment:
maybe delegate this code into a new method? `run_back_forth` (instead of overloading this one ...)